stdlib.ispc

```diff
@@ -3256,7 +3256,7 @@ __declspec(safe) static inline float exp(float x_full) {
 
         float scaled = x_full * one_over_ln2;
         float k_real = floor(scaled);
-        int k = (int)k_real;
+        int k = (int)k_real; // Be careful: float to int convertions are UB if the float value is too huge
```

Comment:
I understand the comment in general, but I don't understand what useful information it brings. It has no call for action - i.e. if the overflow happens, then it happens. It seems there's nothing we can fix here, right? And I don't think we are going to tune this algorithm.

Please correct me if I get this wrong.

Reply:
I added this comment because it was the source of the initial bug : AFAIR we used the `k` variable in a case where it was UB because of the overflow. So I fixed that by using directly `k_real` instead and found useful to mention this UB so to warn future people about this problem (avoiding them to repeat the same mistake again). Maybe it is not needed. I can remove the comment if you think so.

Reply:
I don't think that we are going to hack this algorithm in the future. So this comment belongs more to the commit description, rather than to code itself. Let's remove it.